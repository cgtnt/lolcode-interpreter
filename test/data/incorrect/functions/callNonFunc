HAI

I HAS A ADD ITZ 5

VISIBLE I IZ ADD MKAY

KTHXBYE
