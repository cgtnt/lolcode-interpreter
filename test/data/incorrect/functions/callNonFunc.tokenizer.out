@1:T_BEGIN:HAI-@3:T_COMMAND_TERMINATOR:
-@3:T_DECLARE_VAR:I HAS A-@3:T_T_IDENTIFIER:ADD-@3:T_DECLARE_SET_VAR:ITZ-@3:T_T_INT:5-@5:T_COMMAND_TERMINATOR:
-@5:T_WRITE_STDOUT:VISIBLE-@5:T_FUNC_CALL:I IZ-@5:T_T_IDENTIFIER:ADD-@5:T_END_INF:MKAY-@7:T_COMMAND_TERMINATOR:
-@7:T_END:KTHXBYE-@8:T_COMMAND_TERMINATOR:
-@8:T_EOF:
